test/backend/test.py

```diff
@@ -181,763 +181,569 @@ def supports_device(cls, device):
+    "test_split_equal_parts_default_axis_cpu": test_static_dynamic,
+    "test_split_variable_parts_1d_cpu": test_static_dynamic,
+    "test_split_variable_parts_2d_cpu": test_static_dynamic,
+    "test_split_variable_parts_default_axis_cpu": test_static_dynamic,
     
+    # Sqrt
+    "test_sqrt_cpu": test_static_dynamic,
+    "test_sqrt_example_cpu": test_static_dynamic,
+
+    # Squeeze
+    "test_squeeze_cpu": test_static_dynamic,
+    "test_squeeze_negative_axes_cpu": test_static_dynamic,
+
+    # Str Normalizer
+
+    # Sub
+    "test_sub_cpu": test_static_dynamic,
+    "test_sub_bcast_cpu": test_static_dynamic,
+    "test_sub_example_cpu": test_static_dynamic,
+
+    # Sum
+    "test_sum_example_cpu": test_static_dynamic,
+    "test_sum_one_input_cpu": test_static_dynamic,
+    "test_sum_two_inputs_cpu": test_static_dynamic,
+
+    # Tan
+
+    # Tanh
+    "test_tanh_cpu": test_static_dynamic,
+    "test_tanh_example_cpu": test_static_dynamic,
+
+    # Tfdf Vectorizer
+
+    # Threshold Relu
+
     # Tile
-    #"test_tile_cpu",
-    #"test_tile_precomputed_cpu",
-    
-    # ConstantOfShape
-    "test_constantofshape_float_ones_cpu",
-    "test_constantofshape_int_zeros_cpu",
-    
-    # Less
-    #"test_less_cpu",
-    #"test_less_bcast_cpu",
+    "test_tile_cpu": test_static_dynamic,
+    "test_tile_precomputed_cpu": test_static_dynamic,
+
+    # TopK
+
+    # Training Dropout
+
+    # Transpose
+    "test_transpose_default_cpu": test_static,
+    "test_transpose_all_permutations_0_cpu": test_static_dynamic,
+    "test_transpose_all_permutations_1_cpu": test_static_dynamic,
+    "test_transpose_all_permutations_2_cpu": test_static,
+    "test_transpose_all_permutations_3_cpu": test_static,
+    "test_transpose_all_permutations_4_cpu": test_static,
+    "test_transpose_all_permutations_5_cpu": test_static,
+
+    # Unique
+
+    # Unsqueeze
+    "test_unsqueeze_axis_0_cpu": test_static,
+    "test_unsqueeze_axis_1_cpu": test_static,
+    "test_unsqueeze_axis_2_cpu": test_static,
+    "test_unsqueeze_axis_3_cpu": test_static,
+    "test_unsqueeze_negative_axes_cpu": test_static,
+    "test_unsqueeze_three_axes_cpu": test_static,
+    "test_unsqueeze_two_axes_cpu": test_static,
+    # "test_unsqueeze_unsorted_axes_cpu": test_static,
+
+    # Upsample
+
+    # Where
+
+    # Xor
+
+
+    ############################################################
+    # Model (alphabetical order)
+
+    "test_shufflenet_cpu": test_static,
+    "test_resnet50_cpu": test_static,
+    "test_vgg19_cpu": test_static,
+}
+
+# test for static
+test_to_enable = [ key for (key, value) in test_to_enable_static_dynamic.items() if value & test_static ]
+
+# Specify the test cases which currently can not pass for dynamic shape
+# Presumably, this list should be empty
+# Except for some operation too difficult to handle for dynamic shape
+# or big models
+test_not_for_dynamic = [ key for (key, value) in test_to_enable_static_dynamic.items() if value == test_static ]
```

Comment:
Should test_disabled be considered:  `if value == test_static or value == test_disabled`
Or do not define test_disabled at all.

Reply:
Good point, will update in the next round.

Reply:
But maybe we could switch to a positive logic, add a variable that includes just the tests to run the dynamic, as opposed to list of tests that cannot go dynamic :-)